app/status_test.go

```diff
@@ -30,11 +31,8 @@ func TestSaveStatus(t *testing.T) {
 			th.App.SaveAndBroadcastStatus(status)
 
 			after, err := th.App.GetStatus(user.Id)
-			if err != nil {
-				t.Fatalf("failed to get status after save: %v", err)
-			} else if after.Status != statusString {
-				t.Fatalf("failed to save status, got %v, expected %v", after.Status, statusString)
-			}
+			require.Nil(t, err, "failed to get status after save: %v", err)
+			require.Equal(t, after.Status, statusString, "failed to save status, got %v, expected %v", after.Status, statusString)
```

Comment:
The expected value should be the second parameter, actual is the third parameter.

Reply:
Oh my...I think it is fixed now T_T